planner/core/planbuilder.go

```diff
@@ -3821,13 +3821,13 @@ func (b *PlanBuilder) buildSelectPlanOfInsert(ctx context.Context, insert *ast.I
 	if err != nil {
 		return err
 	}
-	actualColLen := -1
+	actualColLen := insert.ColLen
```

Comment:
Because the initialization value for `insert.ColLen` is 0. And it's impossible for the variable `actualColLen`'s end value be zero(I guess). So we change the `actualColLen`'s initiation value to 0.